### ISSUE TITLE
The name, role, state, and value of all user interface components can be programmatically determined (especially when the state changes).

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#127]
The "Facebook" through "LinkedIn" links under the "Follow Us" heading are missing accessible text alternatives. Provide descriptive text alternatives or accessible labels for each social media link so screen reader users can identify the purpose of the links.

PHP warnings are displayed on the main page content, which may confuse users, distract from the main content, and are read by screen readers.Exposed content should correctly convey information. (Warnings expose irrelevant technical content to users.)

The image + buttons that open a modal don't indicate that they open a modal. Add aria-haspopup="dialog" to the buttons hat trigger a modal.
